packages/html-reporter/src/testFilesView.tsx

```diff
@@ -29,8 +29,9 @@ export const TestFilesView: React.FC<{
   setExpandedFiles: (value: Map<string, boolean>) => void,
   filter: Filter,
   filteredStats: FilteredStats,
+  duration: number,
```

Comment:
It seems like this is always equal to `report?.duration ?? 0`

Reply:
Indeed.